spring-amqp/src/main/java/org/springframework/amqp/support/postprocessor/DeflaterPostProcessor.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2014-2019 the original author or authors.
```

Comment:
Just the current year.
This is fully new class and wasn't created in that `2014` year.